nixos/modules/services/networking/doh-proxy-rust.nix

```diff
@@ -0,0 +1,69 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  cfg = config.services.doh-proxy-rust;
+
+in {
+
+  options.services.doh-proxy-rust = {
+
+    enable = mkEnableOption "doh-proxy-rust";
+
+    flags = mkOption {
+      type = types.listOf types.str;
+      default = [];
+      example = literalExample [ "--server-address=9.9.9.9:53" ];
+      description = ''
+        A list of command-line flags to pass to doh-proxy.
```

Comment:
Including a link to https://github.com/jedisct1/doh-server#usage would be useful.

Reply:
I hope I'm doing this correctly. I've been looking through the docs, but can't figure out how the options index on nixos.org is rendered, and how I would preview this locally. Could you point me in the right direction?

Reply:
Yes that is correct. You can render the manual and take a look with this: `~/nixpkgs (master)> nix-build nixos/release.nix -A manual.x86_64-linux`.